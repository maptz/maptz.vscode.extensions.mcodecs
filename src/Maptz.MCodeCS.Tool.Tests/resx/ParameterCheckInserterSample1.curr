using System;
using System.Threading.Tasks;

namespace MyNamespace{
	
	public class MyClass2
	{
		public IInfo Info { get; set; }
	}

	public class MyClass
	{

		public MyClass(IInfo info)
		{
			
			this.Info = info;
		}

		public IInfo Info { get; set; }

		public Task DoSometihngAsync(IInfo info, string someString, int i)
		{
			var poo = "oiods";
			{{CURSOR}}
		}
	}
}