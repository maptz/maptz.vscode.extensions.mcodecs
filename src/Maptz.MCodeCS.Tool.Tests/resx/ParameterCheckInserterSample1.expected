using System;
using System.Threading.Tasks;

namespace MyNamespace
{
    public class MyClass2
    {
        public IInfo Info
        {
            get;
            set;
        }
    }

    public class MyClass
    {
        public MyClass(IInfo info)
        {
            this.Info = info;
        }

        public IInfo Info
        {
            get;
            set;
        }

        public Task DoSometihngAsync(IInfo info, string someString, int i)
        {
            if (info == null)
            {
                throw new ArgumentNullException("info");
            }

            if (someString == null)
            {
                throw new ArgumentNullException("someString");
            }

            if (i == null)
            {
                throw new ArgumentNullException("i");
            }

            var poo = "oiods";
        }
    }
}